docs/source/migration.mdx

```diff
@@ -900,6 +900,8 @@ https://your.server/?query=%7B__typename%7D
 
 If you want a health check for your HTTP server unrelated to the health of the GraphQL execution engine (i.e., like Apollo Server 3's health check feature), you can add a `GET` handler that always succeeds to your web framework.
 
+To prevent inaccurate positive health checks, ensure that the `Accept` header is either unset or explicitly set to `application/json`. Failure to do so may result in Apollo Server returning an HTML page, especially if the Apollo Sandbox is enabled. It is crucial to note that while Apollo Server 3's health check using trivial query always produces a GraphQL response, Apollo Server v4 will return the HTML if the `Accept` header's value is unset or explicitly set to `application/json`.
```

Comment:
I'd like to break the text up and add a `<Note>` to visually distinguish the second portion. However, the last sentence "Apollo Server v4 will return the HTML if the `Accept` header's value is unset or explicitly set to `application/json`" seems to be at odds with the first two sentences that say, "Failure to [unset or explicitly set...] the `Accept` header causes Apollo Server 4 to return HTML.

Is this an accurate rephrasing?

```suggestion

To prevent inaccurate positive health checks, ensure that the `Accept` header is either unset or explicitly set to `application/json`.

<Note>

Using a trivial query with Apollo Server 3's health check feature always produces a GraphQL response. Apollo Server 4 returns an HTML page unless the `Accept` header's value is unset or explicitly set to `application/json`.

</Note>
```

Reply:
Yes, awesome!!!